packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/EditForm/editForm.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { Form } from 'informed';
+import { func, shape, string, arrayOf } from 'prop-types';
+import { useEditForm } from '@magento/peregrine/lib/talons/CheckoutPage/ShippingInformation/EditForm/useEditForm';
+
+import { mergeClasses } from '../../../../classify';
+import { isRequired } from '../../../../util/formValidators';
+import Button from '../../../Button';
+import Country from '../../../Country';
+import Field, { Message } from '../../../Field';
+import Region from '../../../Region';
+import TextInput from '../../../TextInput';
+import defaultClasses from './editForm.css';
+import EditFormOperations from './editForm.gql';
+
+const EditForm = props => {
+    const { afterSubmit, classes: propClasses, onCancel, shippingData } = props;
+
+    const talonProps = useEditForm({
+        afterSubmit,
+        ...EditFormOperations,
+        shippingData
+    });
+    const { handleSubmit, initialValues, isSaving, isUpdate } = talonProps;
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const messageComponent = !isUpdate ? (
+        <Message>
+            Set a password at the end of guest checkout to create an account in
+            one easy step.
+        </Message>
+    ) : null;
+
+    const cancelButton = isUpdate ? (
+        <Button
+            classes={{
+                root_normalPriority: classes.submit
+            }}
+            disabled={isSaving}
+            onClick={onCancel}
+            priority="normal"
+        >
+            Cancel
+        </Button>
+    ) : null;
+
+    return (
+        <Form
+            className={classes.root}
+            initialValues={initialValues}
+            onSubmit={handleSubmit}
+        >
+            <Field classes={{ root: classes.field }} id="email" label="Email">
```

Comment:
Consider applying these classnames to some wrapping div elements around the fields rather than as classname overrides to the fields themselves.

Reply:
Wrapped in divs to avoid class overrides in 0d829a5.